packages/pwa-kit-create-app/assets/bootstrap/js/package.json.hbs

```diff
@@ -21,7 +21,7 @@
   "bundlesize": [
     {
       "path": "build/main.js",
-      "maxSize": "43 kB"
+      "maxSize": "44 kB"
```

Comment:
Just for my knowledge: Why do we need to increase the maxSize... I have seen this in a number of PR lately

Reply:
the code we wrote bumps the size over the 43kb threshold.

Reply:
This is the limit we defined on the minified javascript file (main.js)